components/table/stories/table.stories.js

```diff
@@ -125,25 +138,156 @@ export default {
 	},
 };
 
+const ExampleRowItems = [
+	{
+		cellContent: ["Table Row Alpha", "Alpha", "Table Row Alpha"],
+		showCheckbox: true,
+	},
+	{
+		cellContent: [
+			"Table Row Bravo. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+			"Bravo",
+			"Table Row Bravo. Lorem ipsum dolor sit amet.",
+		],
+		showCheckbox: true,
+		isSelected: true,
+	},
+	{
+		cellContent: "Table Row Charlie",
+		showCheckbox: true,
+		isSelected: true,
+	},
+	{
+		cellContent: "Table Row Delta",
+		showCheckbox: true,
+	},
+	{
+		cellContent: "Echo",
+		showCheckbox: true,
+	},
+	{
+		cellContent: "Foxtrot",
+		showCheckbox: true,
+	},
+];
+
+/**
+ * The medium size is the default and recommended option.
+ */
 export const Default = TableGroup.bind({});
 Default.args = {};
 
-// ********* VRT ONLY ********* //
-export const WithForcedColors = TableGroup.bind({});
-WithForcedColors.tags = ["!autodocs", "!dev"];
-WithForcedColors.parameters = {
-	chromatic: {
-		forcedColors: "active",
-		modes: disableDefaultModes,
-	},
+// ********* DOCS ONLY ********* //
+/**
+ * Tables come in four different sizes: small, medium, large, and extra-large. The medium size is the default and recommended option.
+ */
+export const Sizing = (args, context) => Sizes({
+	Template,
+	withHeading: false,
+	withBorder: false,
+	...args,
+}, context);
+Sizing.args = {};
+Sizing.tags = ["!dev"];
+Sizing.parameters = {
+	chromatic: { disableSnapshot: true },
 };
 
-// ********* DOCS ONLY ********* //
 /**
- * An example showing both the optional summary row, and a row marked as selected.
+ * The compact variant decreases the spacing used within the table.
  */
-export const SummaryAndSelected = Template.bind({});
+export const Compact = Template.bind({});
+Compact.args = {
+	...Default.args,
+	density: "compact",
+};
+Compact.tags = ["!dev"];
+Compact.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The spacious variant increases the spacing used within the table.
+ */
+export const Spacious = Template.bind({});
+Spacious.args = {
+	...Default.args,
+	density: "spacious",
+};
+Spacious.tags = ["!dev"];
+Spacious.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The standard multi-select table includes a column of checkboxes used for selecting rows.
+ */
+export const MultiSelect = Template.bind({});
```

Comment:
The multiselects and the collapsible stories I _believe_ are just based on the `rowItems` you pass in. They can have separate `isCollapsible` or `showCheckbox` or `isSectionHeader` args that change the look of the table. Is there way for us to do that somehow in the controls, because right now, there's a few stories that cannot be created via controla (multiselects, collapsible, section headers, summary rows, row drop targets).

Reply:
Table is such a huge component, I would think it'd be alright to add a few more stories? Not unlike menu. If it'll add to the number of snapshots, that'd be something to run by @pfulton or @castastrophe though, most likely.

Reply:
https://github.com/adobe/spectrum-css/pull/2925/commits/7dcfb17d4b181da3582bb639005183fc1c1fb225 should add those stories to the sidebar. I still have the `disableSnapshot: true` on, since everything is still captured in the test file, but do I need to do anything else to make sure they're not adding snapshots? I can reach out to Patrick & Cassondra on this too.

Reply:
In that case, I think it's fine like that! We'll just verify that it's not adding any new stories whenever the vrt is run!

I can see there are some stories removed from the sidebar here: Scrollable, Summary and Selected, Thumbnails, and Thumbnails Collapsible. Thumbnails and Scrollable look like they're options, so I think it works that they're removed, but did I miss a control somewhere that makes the summary row show up to display something like the Summary And Selected and Thumbnails Collapsible story?

Reply:
Ah did I miss some? There's so many!!

Summary and Selected- I definitely missed that one. 
Thumbnails Collapsible- oohh this one is weird. You can create this from the collapsible story by toggling on the thumbnails. http://localhost:8080/?path=/story/components-table--collapsible&args=showThumbnails:!true 

After looking into thumbnails collapsible, section header quiet is similar, where you can create it from the section header story, and toggling quiet. http://localhost:8080/?path=/story/components-table--section-header&args=isQuiet:!true

I added them into the sidebar, with the impression that I can't create them via controls from the Default story. https://github.com/adobe/spectrum-css/pull/2925/commits/77cfb20c81ba2412b4eb5fe72bee9b16399612b5